docs/core/install/windows.md

```diff
@@ -264,12 +264,32 @@ While Visual Studio Code doesn't come with an automated .NET Core installer like
 
 The [download page](https://dotnet.microsoft.com/download/dotnet-core) for .NET provides Windows Installer executables.
 
-When you use the MSI files to install .NET< you can customize the installation path by setting the `DOTNETHOME_X64` and `DOTNETHOME_X86` parameters:
+When you use the Windows installers to install .NET, you can customize the installation path by setting the `DOTNETHOME_X64` and `DOTNETHOME_X86` parameters:
 
 ```console
 dotnet-sdk-3.1.301-win-x64.exe DOTNETHOME_X64="F:\dotnet\x64" DOTNETHOME_X86="F:\dotnet\x86"
 ```
 
+If you want to install .NET silently, such as in a production environment or to support continuous integration, use the following switches:
+
+- `/install`\
+Installs .NET.
+
+- `/quiet`\
+Prevents any UI and prompts from displaying.
+
+- `norestart`\
+Suppresses any attempts to restart.
+
+```console
+dotnet-sdk-3.1.301-win-x64.exe /install /quiet /norestart
+```
+
+For more information, see [Standard Installer Command-Line Options](/windows/win32/msi/standard-installer-command-line-options).
```

Comment:
Should we point them to the MSI reference? The commandline options available are what the WiX Engine supports, not MSI.

Reply:
Looks great

Reply:
I thought about pointing them here for the MSI switches: https://docs.microsoft.com/en-us/windows/win32/msi/command-line-options but then I thought it may be confusing since it clearly indicates `msiexec.exe` and the page I did link to is more generic.

Reply:
Although those commands look similar to what the .NET bundles use, both pages apply to ```msiexec```.

Reply:
@joeloff I thought WiX is just a wrapper to configure MSIs during install? Do you have a specific alternative page in mind? Or do you just suggest deleting the link all together?